torchvision/datasets/phototour.py

```diff
@@ -82,8 +82,10 @@ def __init__(
             self.download()
 
         if not self._check_datafile_exists():
-            raise RuntimeError('Dataset not found.' +
-                               ' You can use download=True to download it')
+            try:
+                self.cache()
+            except Exception as error:
```

Comment:
nit: in general it's not ideal to catch generic exceptions like this. I'm merging this to move forward, but it would have been preferable to be a bit more explicit on which exceptions to catch

Reply:
True. Given that I'll remove the functionality soon this was an quick and dirty fix to keep it BC.